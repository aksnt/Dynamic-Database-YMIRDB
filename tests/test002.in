SET a 1 2 3 
SET A 4 5 6 
GET a
GET A
PUSH A 5
POP A
BYE
