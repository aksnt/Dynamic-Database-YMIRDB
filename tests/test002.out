> ok

> ok

> [1 2 3]

> [4 5 6]

> ok

> 5

> [4 5 6]
